koku/masu/external/downloader/gcp/gcp_report_downloader.py

```diff
@@ -297,6 +296,14 @@ def collect_new_manifests(self, current_manifests, bigquery_mappings):
                     "files": [file_name],
                 }
                 new_manifests.append(manifest_metadata)
+        if not new_manifests:
+            stmt = "No new manifests created."
+            extra_context = {
+                "dates": f"{self.scan_start} - {self.scan_end}",
```

Comment:
Separate scan_start and scan_end fields would make searching in the logs easier.
```suggestion
                "scan_start": f"{self.scan_start}",
                "scan_end": f"{self.scan_end}",
```